Tests/Fixtures/smb_scheduled_task_install/fake-powershell.sh

```
@@ -0,0 +1,58 @@
+  local value="${1//\\//}"
+  value="${value#//}"
+  value="${value#*/}"
+  local share="${value%%/*}"
+  value="${value#*/}"
+  [[ "$share" == 'C$' ]]
+  printf '%s/%s' "$SIM_REMOTE_ROOT" "$value"
+}
+
+if [[ -n "${SAS_WORKER_PATH:-}" ]]; then
+  [[ -f "$SAS_WORKER_PATH" ]]
+  ! grep -Fq '$path:' "$SAS_WORKER_PATH"
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)